cli/docs/flags.go

```diff
@@ -43,6 +43,13 @@ const (
 	Poetry = "poetry"
 )
 
+const (
```

Comment:
why not using SubScanType?

Reply:
to seperate the CLI parsing from the logic